proto/import_sstpb.proto

```diff
@@ -221,3 +224,25 @@ message WriteResponse {
     Error error = 1;
     repeated SSTMeta metas = 2;
 }
+
+message DuplicateDetectRequest {
+    kvrpcpb.Context context = 1;
+    bytes start_key = 2;
+    bytes end_key = 3;
+    // Return only the keys found by scanning, not their values.
+    bool key_only = 4;
+    // We only check the data whose timestamp is larger than `min_commit_ts`.
+    uint64 min_commit_ts = 5;
+}
+
+message KvPair {
+    bytes key = 1;
+    bytes value = 2;
+    uint64 commit_ts = 3;
+}
+
+message DuplicateDetectResponse {
+    errorpb.Error region_error = 1;
+    Error key_error = 2;
+    repeated KvPair pairs = 3;
```

Comment:
please add a comment about how the kv pairs are arranged e.g.

```
[
  {key: "key1", value: "value11", commit_ts: 1001},
  {key: "key1", value: "value12", commit_ts: 1004},
  {key: "key1", value: "value13", commit_ts: 1005},
  {key: "key2", value: "value21", commit_ts: 1002},
  {key: "key2", value: "value22", commit_ts: 1004},
  ...
]
```

Reply:
OK.

Reply:
Is the  pairs' order guaranteed？How about add a comment to explain it?

Reply:
Yes. It will be returned in order, but I don’t think we should rely on this assumption